voxelbotutils/cogs/utils/interactions/components/action_row.py

```diff
@@ -26,7 +26,7 @@ def to_dict(self):
         }
 
     @classmethod
-    def from_dict(cls, data: dict):
+    def from_dict(cls, data: list):
```

Comment:
Is this accurate? An ActionRow is itself an object with a components attr.

Reply:
Didn't mean to change that line.